src/ManageCourses.Api/Data/EmailService.cs

```diff
@@ -0,0 +1,42 @@
+
+using System.Net;
+using System.Net.Mail;
+using System.Threading.Tasks;
+
+namespace GovUk.Education.ManageCourses.Api.Data
+{
+    public class EmailService : IEmailService
+    {
+
+        private SmtpClient _smtpClient;
+        private string _user;
+
+        public EmailService(string host, string user, string password)
+        {
+            _user = user;
+            _smtpClient = new SmtpClient
+            {
+                Host = host, // set your SMTP server name here
+                Port = 587, // Port 
```

Comment:
Why hardcoded?

Reply:
I was on the fence about making this a parameter but I want this class to only support port 587, to be RFC compliant. I extracted this as a const to make this choice more explicit